servers/rendering/renderer_rd/shaders/environment/gi.glsl

```diff
@@ -28,13 +28,23 @@ struct ProbeCascadeData {
 	float to_cell; // 1/bounds * grid_size
 };
 
+#ifdef USE_MULTIVIEW
```

Comment:
As I mentioned in the other tweet, I really think we need to do away with arrays and use multiple calls.

Reply:
Changed this around. Let me know what you think.